src/test/java/nl/ordina/jobcrawler/controller/ScraperControllerTest.java

```diff
@@ -18,18 +18,18 @@
     private MockMvc mockMvc;
 
     @MockBean
-    private VacancyStarter vacancyStarter;
+    private ScraperService scraperService;
 
     /**
      * Tests the /scrape end point, if it calls the VacancyStarter scrape method
      */
     @Test
     void scrape() throws Exception {
-        doNothing().when(vacancyStarter).scrape();
+        doNothing().when(scraperService).scrape();
 
         mockMvc.perform(put("/scrape"))
                 .andExpect(status().isOk());
 
-        verify(vacancyStarter, times(1)).scrape();
+        verify(scraperService, times(1)).scrape();
```

Comment:
Build fails due to this line (again?). Might need to look more into what is causing this, or maybe comment out this line of code to let the build pass for now?

Reply:
I agree, I opened request https://github.com/mostafa-221/jobcrawler-backend/pull/42 to disable the test for now or find a solution.